fenix/app/src/main/res/values/strings.xml

```diff
@@ -1844,11 +1844,13 @@
     <!-- Title of the "Manage addresses" screen -->
     <string name="addresses_manage_addresses">Manage addresses</string>
     <!-- The header for the first name of an address -->
-    <string name="addresses_first_name">First Name</string>
+    <string name="addresses_first_name" moz:removedIn="124" tools:ignore="UnusedResources">First Name</string>
     <!-- The header for the middle name of an address -->
-    <string name="addresses_middle_name">Middle Name</string>
+    <string name="addresses_middle_name" moz:removedIn="124" tools:ignore="UnusedResources">Middle Name</string>
     <!-- The header for the last name of an address -->
-    <string name="addresses_last_name">Last Name</string>
+    <string name="addresses_last_name" moz:removedIn="124" tools:ignore="UnusedResources">Last Name</string>
```

Comment:
@issammani These `moz:removedIn` actually need to be bumped to 125 since we are currently in Nightly 125 https://whattrainisitnow.com/. Would you be able to make a followup change? Thanks

cc @MatthewTighe @t-p-white

Reply:
Sorry my bad. In my mind it's still 124 for some reason. Anyways, this PR https://github.com/mozilla-mobile/firefox-android/pull/5776 should fix it 😄